config/src/main/java/org/springframework/security/config/annotation/web/configurers/saml2/Saml2LoginConfigurer.java

```diff
@@ -103,6 +105,14 @@
 
 	private RelyingPartyRegistrationRepository relyingPartyRegistrationRepository;
 
+	private AuthenticationManagerResolver<HttpServletRequest> authenticationManagerResolver;
+
+	public Saml2LoginConfigurer authenticationManagerResolver
```

Comment:
Reminder to add JavaDoc, including a `@since` tag.

Reply:
Thanks. Done. and this method changed from `authenticationManagerResolver` to `authenticationProvider`